deployments/compose/dbt/dbt-project/.user.yml

```diff
@@ -0,0 +1 @@
+id: afae7eda-6594-49b7-b531-ca995407a627
```

Comment:
do we need this one?

Reply:
Generated as part of the `dbt init` command, removing it!